Maestro/subscriptions.json

```diff
@@ -241,6 +276,38 @@
         }
       }
     },
+    // Update dependencies in CoreFX release/2.0.0
+    {
+      "triggerPaths": [
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/release/2.0.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/release/2.0.0/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/coreclr/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectk-tfs/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs/master/Latest.txt",
+        "https://github.com/dotnet/versions/blob/master/build-info/dotnet/sni/master/Latest.txt",
```

Comment:
> "https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectk-tfs/master/Latest.txt",
"https://github.com/dotnet/versions/blob/master/build-info/dotnet/projectn-tfs/master/Latest.txt",
"https://github.com/dotnet/versions/blob/master/build-info/dotnet/sni/master/Latest.txt"

@weshaggard  please could you confirm if these 3 triggers are still required for release/2.0.0.